README.md

```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
```

Comment:
Both spellings exist, but "minuscule" is more common and traditional (its how the Romans spelled it).